DigitalLearningSolutions.Web/Views/VerifyEmail/Index.cshtml

```diff
@@ -0,0 +1,130 @@
+﻿@using DigitalLearningSolutions.Data.Enums
+@using DigitalLearningSolutions.Web.ViewModels.VerifyEmail
+@model VerifyEmailViewModel
+
+@{
+  ViewData["Title"] = "Verify your email address";
+}
+
+@section NavMenuItems {
+  <partial name="_NavMenuItems" />
+}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full">
+    <h1 class="nhsuk-heading-xl">Verify your email address</h1>
+
+    @if (Model.UnverifiedEmailsCount == 0) {
+      <p class="nhsuk-body-m"> All your email addresses are verified.</p>
+    } else {
+      var centreEmailsExcludingFirstParagraph = (Model.PrimaryEmail == null
+        ? Model.CentreSpecificEmails.Skip(1)
+        : Model.CentreSpecificEmails).ToList();
+      const string centreEmailExplanation = "You will not be able to access that account until you verify the address.";
+
+      @if (Model.EmailVerificationReason.Equals(EmailVerificationReason.EmailNotVerified)) {
+        string primaryEmail;
+        string emailAddresses;
+        string link;
+        string address;
+        string email;
+        string folder;
+        string pronoun;
+
+        @if (Model.UnverifiedEmailsCount == 1) {
+          primaryEmail = null;
+          emailAddresses = Model.PrimaryEmail ?? "this address";
+          link = "link";
+          address = "address";
+          email = "this email";
+          folder = "folder";
+          pronoun = "it";
+        } else {
+          primaryEmail = $", {Model.PrimaryEmail},";
+          emailAddresses = "each of the addresses above";
+          link = "links";
+          address = "addresses";
+          email = "the emails";
+          folder = "folders";
+          pronoun = "them";
+        }
+        @if (Model.PrimaryEmail != null) {
+          var primaryEmailText = $"Your primary email address{primaryEmail} has not been verified yet." +
+                                 " You can edit your account details, but you cannot access any centre accounts until it is verified.";
+
+          <p class="nhsuk-body-m">@primaryEmailText</p>
```

Comment:
I know the wireframe has the phrasing so that it needs this string interpolation stuff, but I think it would be OK to always just have the ', Model.PrimaryEmail,' chunk of text, even if it is the only email to be verified.
We'd have to confirm that HEE are OK with the final wording, but since we've already changed some wording, we probably need to do that anyway.